lib/inspector.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const connect = process.binding('inspector').connect;
+const EventEmitter = require('events');
+
+if (!connect)
+  throw new Error('Inspector is not available');
+
+const connectionSymbol = Symbol('connectionProperty');
+const messageCallbacksSymbol = Symbol('messageCallbacks');
+const nextIdSymbol = Symbol('nextId');
+const onMessageSymbol = Symbol('onMessage');
+
+class Session extends EventEmitter {
+  constructor() {
+    super();
+    this[connectionSymbol] =
+        connect((message) => this[onMessageSymbol](message));
```

Comment:
Should we require an explicit `connect` call rather than connecting by default in the constructor? Once one connection has been made no further connections can be made, which might surprise people creating another new Session.

Also, if the session is disconnected with `disconnect` there's no way to reconnect it.